grader/settings.py

```diff
@@ -9,7 +9,7 @@
 
 # Base options, commonly overridden in local_settings.py
 ##########################################################################
-DEBUG = False
+DEBUG = True
```

Comment:
You can set `GRADER_DEBUG=true` environment variable for the container. Any case, this should stay as false.